CHANGELOG.md

```diff
@@ -2,10 +2,55 @@
 
 ## Unreleased
 
-- [Pull request #601: Add input classes to implement a part of #428](https://github.com/alphagov/accessible-autocomplete/pull/601)
-- [Pull request #621: Handle query change when the text length does not change](https://github.com/alphagov/accessible-autocomplete/pull/621)
-- [Pull request #620: Add `className` attribute to status component](https://github.com/alphagov/accessible-autocomplete/pull/620)
-- [Pull request #591: Add menuAttributes to fix #361](https://github.com/alphagov/accessible-autocomplete/pull/591)
+### Breaking changes
+
+#### Verify your code does not rely on removed polyfills
+
+Following on from [our previous announcement](https://github.com/alphagov/design-system-team-internal/issues/561), the accessible autocomplete code is no longer transpiled and polyfilled to support IE8-10.
+
+However, because the polyfills create or extend global objects, you might have other code in your service unintentionally relying on the inclusion of these polyfills. You might need to introduce your own polyfills or rewrite your JavaScript to avoid using the polyfilled features.
+
+This change was introduced in [pull request #612: Update packages, configs + Node.js 20](https://github.com/alphagov/accessible-autocomplete/pull/612).
+
+#### Check for minor visual changes in the rendering of the component
```

Comment:
is it a particular component? if not, then this possibly needs to say 'a component' or 'the components' or 'any component' depending on what is the most accurate.

Reply:
Yes, the 'the' refers to the Accessible autocomplete component. It's the only component in this repository, so I think 'the' would be the most accurate in this case.

Reply:
ok. then you can ignore all my suggestions. I'll see if i can remove/resolve them